.github/workflows/build_wasm.yml

```diff
@@ -0,0 +1,71 @@
+name: build_linux
```

Comment:
As a note let's call this something new (like build_wasm) instead of build_linux ( as the builds will go into the same area as build_linux) and as discussed let's leave the existing buid_linux and build_mac as is. Thanks.